sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -107,3 +109,28 @@ def test_check_estimators_unfitted():
     # check that CorrectNotFittedError inherit from either ValueError
     # or AttributeError
     check_estimators_unfitted("estimator", CorrectNotFittedErrorClassifier)
+
+
+def test_common():
+    # check that the fit function of an estimator will accept
+    # a 'sample_weight' parameter of type pandas.Series
+    try:
+        import pandas as pd
+    except ImportError:
+        raise SkipTest("Pandas not found")
+    X = pd.DataFrame([[1, 1], [1, 2], [1, 3], [2, 1], [2, 2], [2, 3]])
+    y = pd.Series([1, 1, 1, 2, 2, 2])
+    weights = pd.Series([1] * 6)
+    for estimator_name, Estimator in all_estimators():
```

Comment:
This should be implemented like the other checks in `sklearn.utils.estimator_checks`, not as a separate test.